lsst/ts/laser/csc.py

```diff
@@ -1,25 +1,34 @@
-"""Implements CSC classes.
+"""Implements CSC classes for the TunableLaser.
 
 """
 import logging
 from lsst.ts.laser.component import LaserComponent
-import salobj
+from lsst.ts.salobj import *
```

Comment:
A late comment! I think `flake8` will complain about this but I'd rather do 

```python

from lsst.ts import salobj
```

I'm not sure it is recommended to do `import *`... 

I've seen this also on other places which I recommend are changed as well...